test/SocioCatClaim.t.sol

```diff
@@ -117,6 +120,46 @@ contract SocioCatClaimTest is Test {
         vm.stopPrank();
     }
 
+    function test_setSigner() public {
+        (address newSigner, ) = makeAddrAndKey("newSigner");
+        // --
+
+        vm.startPrank(owner);
+        claim.setSigner(newSigner);
+        vm.stopPrank();
+
+        assertEq(claim.signer(), newSigner);
+    }
+
+    function test_setSigner_rejectsZeroSigner() public {
+        vm.expectRevert(SocioCatClaim.ZeroAddress.selector);
+
+        vm.startPrank(owner);
+        claim.setSigner(address(0));
+        vm.stopPrank();
+    }
+
+    function test_setSigner_rejectsNotOwner() public {
+        (address newSigner, ) = makeAddrAndKey("newSigner");
+        // --
+
+        vm.expectRevert();
```

Comment:
specify revert selector here, make sure it revert `OwnableUnauthorizedAccount`

Reply:
Done